docs/src/How To Work With Us.doc.js

```diff
@@ -0,0 +1,120 @@
+// @flow strict
+import React, { type Node } from 'react';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+
+const card = (c) => cards.push(c);
+
```

Comment:
I don't see the PageHeader here anymore :(

Reply:
Ugh, fixing.